### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by CvX**

### CODE REVIEW

**Comment by fsmanuel** [comment]
@CvX thanks for the PR. I'll review it now.

File: .travis.yml [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
-  - EMBER_TRY_SCENARIO=ember-beta
-  - EMBER_TRY_SCENARIO=ember-canary
-  - EMBER_TRY_SCENARIO=ember-default
+  global:
+    # See https://git.io/vdao3 for details.
+    - JOBS=1
+  matrix:
+    # we recommend new addons test the current and previous LTS
+    # as well as latest stable release (bonus points to beta/canary)
+    - EMBER_TRY_SCENARIO=ember-lts-2.8
```
Comment (fsmanuel) [inline .travis.yml]: Can we have this list:
```
    - EMBER_TRY_SCENARIO=ember-lts-2.8
    - EMBER_TRY_SCENARIO=ember-lts-2.12
    - EMBER_TRY_SCENARIO=ember-lts-2.16
    - EMBER_TRY_SCENARIO=ember-release
    - EMBER_TRY_SCENARIO=ember-beta
    - EMBER_TRY_SCENARIO=ember-canary
    - EMBER_TRY_SCENARIO=ember-default
```

File: .travis.yml [hunk 2]
```diff
@@ -43,8 +45,4 @@ install:
 script:
   # Usually, it's ok to finish the test scenario without reverting
   #  to the addon's original dependency state, skipping "cleanup".
-  - node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO test --skip-cleanup
-
-branches:
```
Comment (fsmanuel) [inline .travis.yml]: Can you readd it?

File: package.json [hunk 3]
```diff
@@ -28,40 +28,39 @@
...
+    "ember-cli-version-checker": "^2.1.0"
   },
   "devDependencies": {
     "broccoli-asset-rev": "^2.5.0",
     "ember-ajax": "^3.0.0",
-    "ember-cli": "^2.14.0",
-    "ember-cli-dependency-checker": "^1.4.0",
-    "ember-cli-eslint": "^3.1.0",
-    "ember-cli-github-pages": "0.1.2",
+    "ember-cli": "~2.16.2",
```
Comment (fsmanuel) [inline package.json]: Can you update to `2.17.0` which got released 2 days ago?

File: package.json [hunk 4]
```diff
@@ -28,40 +28,39 @@
...
+    "broccoli-funnel": "^2.0.1",
+    "broccoli-merge-trees": "^2.0.0",
     "broccoli-stew": "^1.5.0",
-    "chalk": "^1.1.1",
-    "ember-cli-babel": "^6.3.0",
+    "chalk": "^2.3.0",
+    "ember-cli-babel": "^6.6.0",
     "ember-cli-string-utils": "^1.1.0",
-    "ember-cli-version-checker": "^1.3.1",
-    "ember-getowner-polyfill": "^1.2.3"
```
Comment (fsmanuel) [inline package.json]: I think we should leave `ember-getowner-polyfill` for older versions?!

Comment (CvX) [reply package.json]: Depends on which ember versions is ember-local-storage currently supporting. Only 2.4+ are being tested, and that polyfill is needed only for <2.3, so we should either test e.g. 2.0, or decide that only 2.4+ are supported now and bump major version.

File: .travis.yml (hunk 1 above)
Comment (CvX) [reply .travis.yml]: Done, and also re-added 2.4, since I don't think there's any reason not to support it.

File: .travis.yml (hunk 2 above)
Comment (CvX) [reply .travis.yml]: My bad, missed it when doing the diffing on `ember init`. Re-added.

File: package.json (hunk 3 above)
Comment (CvX) [reply package.json]: Done, cli and other dependencies are up to date again.

File: package.json (hunk 4 above)
Comment (fsmanuel) [reply package.json]: Good point. No let's drop support for < 2.3

**Comment by fsmanuel** [comment]
@CvX After CI is green this is good to go. Thanks a lot!

**Comment by CvX** [comment]
Sure thing! After it gets merged I'll rebase #273, run the tests, and re-open it.

**Pull request merged**